static/templates/settings/realm-domains-modal.handlebars

```diff
@@ -14,14 +14,22 @@
             </tbody>
             <tfoot>
                 <tr id="add-realm-domain-widget">
-                    <td><input type="text" class="new-realm-domain" placeholder="acme.com"></td>
-                    <td><input type="checkbox" class="new-realm-domain-allow-subdomains"></td>
+                    <!-- this doesn't need to be translated but it's better than including an exclude rule. -->
+                    <td><input type="text" class="new-realm-domain" placeholder="{{t "acme.com" }}" /></td>
```

Comment:
You don't need to do this; just update `tools/linter_lib/custom_check.py` like this:
```
-                           '<td><input type="text" class="new-realm-domain" placeholder="acme.com"></td>'),
+                           '<td><input type="text" class="new-realm-domain" placeholder="acme.com" /></td>'),
```